test/lib/buildFactory.test.js

```diff
@@ -134,30 +134,30 @@ describe('Build Factory', () => {
         const isoTime = (new Date(dateNow)).toISOString();
         const container = 'node:4';
         const steps = [
-            { name: 'sd-setup-launcher' },
-            { name: 'sd-setup-scm', command: 'git clone' },
-            { command: 'npm install', name: 'init' },
-            { command: 'npm test', name: 'test' }
+            { name: 'sd-setup-launcher', alwaysRun: true },
+            { name: 'sd-setup-scm', command: 'git clone', alwaysRun: true },
+            { command: 'npm install', name: 'init', alwaysRun: false },
```

Comment:
It does actually default to `false` in `buildFactory.js` itself (https://github.com/screwdriver-cd/models/blob/alwaysrun/lib/buildFactory.js#L161), but these are just the expected steps (test data).